config/config.go

```diff
@@ -481,6 +481,8 @@ type GlobalConfig struct {
 	KeepDroppedTargets uint `yaml:"keep_dropped_targets,omitempty"`
 	// Allow UTF8 Metric and Label Names.
 	MetricNameValidationScheme string `yaml:"metric_name_validation_scheme,omitempty"`
+	// Metric name escaping mode to request through content negotiation.
+	MetricNameEscapingScheme string `yaml:"metric_name_escaping_scheme,omitempty"`
```

Comment:
Do you mind updating docs (https://github.com/prometheus/prometheus/blob/efa4f4a60bee54d5263c24221bbbc4cddf0a5d6a/docs/configuration/configuration.md) with this new field? Do you mind also adding possible values and their consequences?